plugins/docker.yml

```diff
@@ -0,0 +1,5 @@
+# Plugins loaded by https://github.com/loomio/loomio/blob/master/Dockerfile
+plugins:
```

Comment:
This wants to mirror https://github.com/loomio/loomio/blob/master/config/plugins.yml
(we've dropped the top level 'plugins' line)